charts/galoy/templates/api-ingress.yaml

```
@@ -15,6 +15,9 @@ metadata:
 
     nginx.ingress.kubernetes.io/proxy-read-timeout: "3600" # 1 hour
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600" # 1 hour
+    nginx.ingress.kubernetes.io/proxy-connect-timeout: "5s"
```

5 seconds feels very long here - I would think that it should typically respond within < 1000ms. They are on the same local network.

Perhaps check honeycomb for 99 percentile latency for this hop and choose 1.5x to 2x that.

---

The effect of the changes are same in my local dev environment with:
```suggestion
    nginx.ingress.kubernetes.io/proxy-connect-timeout: "1s"
```

---

The P99 latency is 1400ms in honeycomb. i just checked the home page, not sure if this is the right spot to check it. so `2s`  might be a good value to test @openoms

---

i think the .1 second 502 makes sense, its probably the nginx rate limiting settings kicking in